frontend/packages/ux-editor/src/components/TextResourceEdit.tsx

```diff
@@ -81,6 +81,7 @@ const TextBox = ({ language, t, textResource, textResourceId }: TextBoxProps) =>
   return (
     <div>
       <TextArea
+        id={classes.textArea}
```

Comment:
Bruk `className` her. `id` skal være unik, og man skal kunne ha flere av denne komponenten på samme side. Foreslår at du også legger til `resize='vertical'` for å gjøre det mulig å utvide tekstfeltet etter behov.

Reply:
Alternativt kan du legge til f.eks. `rows={5}` i stedet for å styre høyden med CSS.

Reply:
I fjernet Id og brukte (rows)

Reply:
lagt til resize